build-tools/automation/azure-pipelines.yaml

```diff
@@ -409,6 +390,27 @@ stages:
         artifactFolder: $(DotNetTargetFramework)-AotLlvm
         useDotNet: true
 
+    - template: yaml-templates/apk-instrumentation.yaml
+      parameters:
+        configuration: $(XA.Build.Configuration)
+        testName: Xamarin.Android.BindingRuntime_Tests
+        project: tests/BindingTests/Xamarin.Android.BindingRuntime-Tests/Xamarin.Android.BindingRuntime-Tests.csproj
+        testResultsFiles: TestResult-Xamarin.Android.BindingRuntime_Tests-$(ApkTestConfiguration).xml
+        artifactSource: bin/Test$(XA.Build.Configuration)/$(DotNetTargetFramework)-android/Xamarin.Android.BindingRuntime_Tests-Signed.apk
+        artifactFolder: $(DotNetTargetFramework)-Default
+        useDotNet: true
+
+    - template: yaml-templates/apk-instrumentation.yaml
+      parameters:
+        configuration: $(XA.Build.Configuration)
+        testName: Xamarin.Android.BindingRuntime_Tests_FastDev
+        project: tests/BindingTests/Xamarin.Android.BindingRuntime-Tests/Xamarin.Android.BindingRuntime-Tests.csproj
+        testResultsFiles: TestResult-Xamarin.Android.BindingRuntime_Tests_FastDev-$(ApkTestConfiguration).xml
```

Comment:
For some reason the `$(ApkTestConfiguration)` is only set on the classic APK test job, I would replace this with `$(XA.Build.Configuration)` instead. I think this is preventing the [test results from being uploaded](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=7343675&view=logs&j=5ded96ba-e325-5baf-fde0-f1868aa3be52&t=113564c2-5517-5674-66e2-044addc8c3ea&l=11):

> ##[warning]No test result files matching TestResult-Xamarin.Android.BindingRuntime_Tests-$(ApkTestConfiguration).xml were found.